src/main/webapp/jsp/notification.jsp

```diff
@@ -0,0 +1,36 @@
+<%@ page import="java.io.PrintWriter" %>
+<%@ page import="java.util.Map" %>
+<%@ page import="com.todolist.app.model.Item" %>
+<%@ page import="java.util.Optional" %>
+<%@ page import="com.todolist.app.service.ToDoStoreSingleton" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Notification</title>
+</head>
+<body>
+<%
+    ToDoStoreSingleton singletonStore = ToDoStoreSingleton.getInstance();
```

Comment:
Could you adjust this code of jsp page. I mean put not the whole java code. E.g. https://www.geeksforgeeks.org/getattribute-passing-data-from-server-to-jsp/